portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -25,6 +25,15 @@ function deleteAllCommentsFromStorage() {
 	}).then(() => this.fetchAndInsertDataIntoDOM());
 }
 
+/*
+ * Deletes a single comment from Datastore by id
+ */
+function deleteCommentFromStorageById(id) {
+	fetch(`delete-data?id=${id}`, {
+		method: 'POST'
+	}).then(() => this.fetchAndInsertDataIntoDOM());
```

Comment:
This works, a less expensive operation might be to simply remove the comment in the UI.

Reply:
That's a good point, I'll look into that.